pom.xml

```diff
@@ -112,42 +112,49 @@
       <groupId>org.apache.pulsar</groupId>
       <artifactId>pulsar-websocket</artifactId>
       <version>${pulsar.version}</version>
+      <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>org.apache.pulsar</groupId>
       <artifactId>pulsar-broker</artifactId>
       <version>${pulsar.version}</version>
+      <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>org.apache.pulsar</groupId>
       <artifactId>testmocks</artifactId>
       <version>${pulsar.version}</version>
+      <scope>provided</scope>
```

Comment:
Perhaps this could be changed to `test` scope in this case though.

Reply:
Yes